### ISSUE TITLE
Review Day Missions Collection in Production

### ISSUE THREAD

**Issue by motropuk** [yalelibrary/YUL-DC#1912]
**Story**

Following on from #1785, collection was moved into Production but has multiple issues that need to be reviewed and fixed. Moving this work to a new ticket here

Review work spreadsheet https://docs.google.com/spreadsheets/d/1kcxYqDlVwlqk4WZf_d3Cje4HAKXHiCEtevH3Sn3c8HU/edit?usp=sharing  

Issues
6362 on front end in Findit
6342 in Production DCS management
6230  in Production Blacklight  https://collections.library.yale.edu/catalog?f%5bproject_identifier_tesi%5d%5b%5d=28&q=&search_field=all_fields
Diff of 112 between DCS management and blacklight
Diff of 20 between Findit and DCS management app

**Acceptance**
- [x] Audit objects in DCS PROD
- [x] Follow up with Stakeholder
- [x] Fix problems
- [x] Final audit to ensure numbers between Findit and DCS match up
-  [x] Update handles URLs to point to DCS

**Comment by motropuk** [comment]
Report outlining objects not currently displaying in DCS is now complete https://docs.google.com/spreadsheets/d/1kcxYqDlVwlqk4WZf_d3Cje4HAKXHiCEtevH3Sn3c8HU/edit#gid=0. We have send this to collection owner to ask how they would like to proceed. Waiting for response

**Comment by motropuk** [comment]
Going to move this ticket to on hold as we have been waiting for a while now

**Comment by motropuk** [comment]
Putting this back in Sprint as we got a response from Divinity

**Comment by sshetenhelm** [comment]
[Updated list of issues and statuses](https://docs.google.com/spreadsheets/d/1YDKBnpY8C8rBI1UnmAenuPX5errm2mKbaUG0mZBF_hQ/edit?usp=sharing) (apologies for google sheet, file upload not working) 

We are waiting on Graziano to provide us with files for 19 parents with 636 missing children.

I am working on confirming the 101 parents with related children he has sent us.

**Comment by sshetenhelm** [comment]
78 parents out of 101 ready for DCS -- @MaggieZhaoYale they are in //storage.yale.edu/home/FC_YULDCS-807001-YUL > DCS_MaggieandSummer > Day Missions Migration > READY

Other 23 parents have issues (received wrong file, more children sent than in MGMT, etc.). Will coordinate with Graziano.

**Comment by sshetenhelm** [comment]
@MaggieZhaoYale George put in captions in Ladybird for parent objects 16024279, 16387457, 16387556, 16387865, 16388118, and 16864427, but they are not published to FindIt. Can we bring these children into DCS now that they have LB captions, or do they need to be published to FindIt? I have the image files, so as long as we can get the child OIDs in, that would be great.

**Comment by sshetenhelm** [comment]
@MaggieZhaoYale 
94 parents are ready to have their children image files added. Files are in //storage.yale.edu/home/FC_YULDCS-807001-YUL > DCS_MaggieandSummer > Day Missions Migration > READY

The follow parents are NOT in LB, but we have all the files. They need new child OIDs:
11126125, 11394586, 11431593, 16388279, 16804878, 16804879, 16804880, 16804881, 16804882, 16804929, 17126607
Image files are available at DCS_MaggieandSummer > Day Missions Migration > Need New Child OIDs - Not in LB

Parents 11093168 and 11094657 are still missing files. Asking for files.

OID 16755882 only has 85 files in LB, but Div has given us 130. Asking about discrepancy.

**Comment by sshetenhelm** [comment]
@MaggieZhaoYale we have received files for the remaining three parents: 11093168, 11094657, and 16755882 

They are in DCS_MaggieandSummer > Day Missions Migration > READY final parents

**Comment by MaggieZhaoYale** [comment]
@sshetenhelm Replaced 1627 images. 2 Images failed. 
Please ignore the third failed and 2 in progress which were fixed, however the message did not update. 
 16387787 https://collections.library.yale.edu/management/batch_processes/8769/parent_objects/16387791
11068498 https://collections.library.yale.edu/management/batch_processes/8769/parent_objects/11068468
The two failed images both have issue by checking with imageMagick.
If the two images can be replaced.

**Comment by sshetenhelm** [comment]
@MaggieZhaoYale Those two children you've linked are showing up for me, and their parents are active in public-facing DCS :)

**Comment by sshetenhelm** [comment]
Here are all the outstanding issues summarized: 

**Attach Image(s) to Existing MGMT Children**
The following OIDs need images attached to children in FindIt. They are as follows: 
11076788
11085492
11108359
11108422
16854157
16755882

**Missing Children in MGMT, But Have All Children in FindIt/Ladybird**
The following OIDs have no children in MGMT but have visible children in both Ladybird and FindIt. Is is possible to bring the children over with their OIDs? 

16864427
16024279

Once we have the empty children with OIDs in MGMT, I can rename the image files with their child OIDs and share them.

**No Children in MGMT and Not in Ladybird**
The followings OIDs have no children in MGMT and no Ladybird records. They will need new child OIDs. The OIDs and number of files we received from DIV are below:

11126125 - 10
11394586 - 52
1143159 - 128
1680487 - 64
1680487 - 51
16804880 - 54
16804881 - 48
16804882 - 46
16804929 - 47
17126607 - 114

**Missing Children in MGMT that are in Ladybird**
These OIDs are missing children that appear in Ladybird 
16387457
16387556
16387865
16388118
16388279
![Screen Shot 2022-05-25 at 2.01.40 PM.png](https://images.zenhubusercontent.com/61f05050155a1d4a7085c8b9/21c690e2-fe23-474b-bc27-e4eccb62c4af)
Sorry for the image, .csv update wasn't happening

**Comment by sshetenhelm** [comment]
@MaggieZhaoYale I added 16388279 to the folder 'Captions Added in Ladybird' 

The other two parents are "Duplicate in Ladybird but not in FindIt" -- can we bring in the OIDS 16030393 and 16819441 from Ladybird? 

Working on compiling some other stats, will update hopefully before standup 06.02

**Comment by sshetenhelm** [comment]
I *think* I've found all the missing parents between FindIt and PROD MGMT, although it brings our PROD MGMT number up to 6,367, and there are only 6,362 objects in FindIt. 
OID list here: [https://app.zenhub.com/files/242185115/98e674fd-7200-4f8a-8884-14d9c037f002/download](https://app.zenhub.com/files/242185115/98e674fd-7200-4f8a-8884-14d9c037f002/download)

@MaggieZhaoYale could you try and get the parents on that list from Ladybird into DCS?

**Comment by MaggieZhaoYale** [comment]
Added 16388279

**Comment by sshetenhelm** [comment]
@MaggieZhaoYale f
iles for OID 16030393 and OID 16819441 are in DCS_MaggieandSummer > Day Missions Migration > Ready 2022 06 06

Files are named with original child OIDs

**Comment by MaggieZhaoYale** [comment]
@sshetenhelm https://collections.library.yale.edu/management/batch_processes/8857 16030393 and OID 16819441 are in the DCS now

**Comment by sshetenhelm** [comment]
14 of 25 completed, working through last 11

**Comment by sshetenhelm** [comment]
6362 on front end in Findit
6369 in Production DCS management
6365 in Production Blacklight

In the process of reconciling these numbers.

**Comment by sshetenhelm** [comment]
Errors discovered in Reconciliation: 

- 4 parents are broken in FIndIt. I will get the missing files.
- 5 parents are in FindIt, but not in DCS.
- 10 parents are duplicated in FindIt, but not in DCS. We should subtract 10 from the FindIt total.
- 33 objects are in DCS, but not in FindIt. I will confirm with Graziano that these should be included.
- 10 objects are either missing from DCS or duplicated in FindIt. I will confirm with Graziano what should be included.

**Comment by sshetenhelm** [comment]
- DIV has confirmed that the 33 objects in DCS but not FindIt should be in DCS. These objects are good to go.
- DIV has confirmed that 10 objects are duplicated in FindIt and do NOT need to be moved to DCS.
- DIV has confirmed that 2 of the objects thought to be missing from DCS are duplicated in FindIt and do NOT need to be moved to DCS.

We need to complete the following:

**Replacement Image Files** 
- [ ] Add one replacement child image for parent 11522888
- [ ] Add one replacement child image for parent 11680050
- [ ] Add two replacement child images for parent 11093521

These images are in DCS_MaggieandSummer > Day Missions Migration > Replacement Images

DIV can't find replacement files for OID 11466215, so they will rescan and provide file later. 

**Add to PROD MGMT**
- [ ] 16553994 - active children in FindIt
- [ ] 16966179 - will need child OIDs
- [ ] 16966178 - will need child OIDs

These images are in DCS_MaggieandSummer > Day Missions Migration > Add Parents to DCS

**Comment by MaggieZhaoYale** [comment]
@sshetenhelm 
Replaced the images: https://collections.library.yale.edu/management/batch_processes/9093
Added 16553994 to Prod https://collections.library.yale.edu/management/batch_processes/9092
16966179 and 16966178:
 -  Finished: 
     - generated new oids for the children
     - rename the ladybird images with new oids
     - added the images to the image pairtree
     - create a csv for the reassociate child oid (cannot attach a csv file, will send it to via email)

   To do: attach each child to the parent.

**Comment by MaggieZhaoYale** [comment]
[https://app.zenhub.com/files/242185115/b060c962-7153-472e-8967-cceceb131a26/download](https://app.zenhub.com/files/242185115/b060c962-7153-472e-8967-cceceb131a26/download)(only 6352 records in the handle, I will check if there are missing records)

**Comment by sshetenhelm** [comment]
DONE!!!! Ready to close!!! :) :) :)
